packages/headless/src/commerce.index.ts

```diff
@@ -71,7 +71,6 @@ export type {
   CartProps,
   Cart,
   CartState,
-  CartControllerState,
```

Comment:
Why do we not expose this anymore?

Reply:
It was defined as 

```
export type CartControllerState = Cart['state'];
```

i.e., `CartState`

So I don't really see the need for it if we export `CartState`.